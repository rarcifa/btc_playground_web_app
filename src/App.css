.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.react-tabs__tab-list {
  border-bottom: 1px solid rgb(255, 255, 255) !important;
  margin: 0 0 10px;
  padding: 0;
}

.App-header {
  background-color: #ffffff;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: black;
}

button {
  appearance: auto;
  text-rendering: auto;
  color: -internal-light-dark(black, white);
  letter-spacing: normal;
  word-spacing: normal;
  text-transform: none;
  text-indent: 0px;
  text-shadow: none;
  display: inline-block;
  text-align: center;
  align-items: flex-start;
  cursor: default;
  background-color: rgba(255, 255, 255, 0);
  box-sizing: border-box;
  font: 400 13.3333px Arial;
  padding: 0;
  border-width: 2px;
  border-style: outset;
  border-color:rgba(255, 255, 255, 0);
  border-image: initial;
}


.MuiButton-outlinedPrimary {
  color: #3f51b5;
  border: 1px solid rgba(63, 81, 181, 0.5);
  border-color:#3f51b5;
}
.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}