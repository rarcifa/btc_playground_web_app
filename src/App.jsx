import React from 'react';
import './App.css';
import MethodOne from './components/MethodOne';
import MethodTwo from './components/MethodTwo';
import MethodThree from './components/MethodThree';
import Footer from './components/FooterComponent'
import Header from './components/HeaderComponent'
import InputList from './components/InputList/InputList'

// Main App 
function App() {

  return (
    <div className="App">

      <header className="App-header">

        {/** Header Title */}
        <Header />

        {/** First Method */}
        <MethodOne />

        {/** Second Method */}
        <MethodTwo />

      {/** Third Method */}
        <InputList/>

        {/** Automation  Method */}
        <MethodThree />

      </header>

      {/** Footer */}
      <Footer/>
    </div>
  );
}

export default App;
