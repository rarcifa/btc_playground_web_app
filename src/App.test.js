import React from 'react';
import { mount, shallow } from 'enzyme';
import { MuiThemeProvider } from '@material-ui/core';
import App from './App';
import theme from './theme';

const props = {};

const wrapper = mount(
  <MuiThemeProvider theme={theme}>
    <App {...props}/>
  </MuiThemeProvider>
);

describe('App', () => {
  it('should render the App Component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});