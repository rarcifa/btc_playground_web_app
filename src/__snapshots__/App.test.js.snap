// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component correctly 1`] = `ReactWrapper {}`;
